pkg/curatedpackages/testdata/values_empty.yaml

```diff
@@ -3,6 +3,7 @@ registryMirrorSecret:
   username: ""
   password: ""
   cacertcontent: ""
+  insecure: "ZmFsc2U="
```

Comment:
This should only true or false, right? Should we just check for if it is set to true and otherwise default to false.

Reply:
The handling on our side should only accept true, but I think it is a bit more clear to set it to false rather than not set it at all.

Reply:
But currently it looks like as long as this field is set, we set it to true which doesn't seem like it would make sense. I think it is better to check for the true or t explicit fields or otherwise assume it is false so let it be false.

Reply:
I'm not following.
```
 % echo 'ZmFsc2U=' | base64 -d
false%                                            
```

Reply:
oh I see so it is base64 encoded.